packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -2300,7 +2310,7 @@ export enum CoreDnsComputeType {
   /**
    * Deploy CoreDNS on Fargate-managed instances.
    */
-  FARGATE = 'fargate'
+  FARGATE = 'fargate',
```

Comment:
This is an unrelated change, I think revert it. While it is nice to fix the styling, it will mess up the git blame. When new entries are actually added to this enum, someone can do it then.

Same comment for the other 2